### DISCUSSION TITLE
Sending feedback in Github mobile app doesn't work

### DISCUSSION THREAD

**Discussion by ceskyDJ** [community/community#64296] [Mobile]
### Select Topic Area

Bug

### Body

Hi,

I found a problem with sending feedback in an official Android app. See: https://www.loom.com/share/5fdea965226e43bfbc078db4c15f133b.

HW/SW details:

- Device: Samsung Galaxy S22
- OS: Android 13
- App version: 1.126.0

**Comment by aconsuegra** [comment]
Hi @ceskyDJ, thanks a lot fore reaching out, and I'm sorry about the inconveniences. I have been testing the flow myself and couldn't reproduce the issue, so it could be related to the content of the message you are trying to send, or something else, but I would appreciate your help debugging this.

Does it happen always to you? Or only sometimes? Could you try removing the URL from the message? Could you try removing the signature?

**Reply by ceskyDJ** [reply]
Hi, I don't test it more. I can, but after August 22nd. So, I'll try to send a few reports and share how it happened. You can ignore those reports.

**Reply by ceskyDJ** [reply]
It looks like it doesn't work at all for me. As you need exact and professional**-like** (see warning at the end) report, it is attached below. I hope it's enough for you. If not, I think I can't help you more. If you need any other **concrete** information, give me a word.

## Issue description

When I try to send any (even minimal) bug report in GitHub Android app, it does nothing except showing error message "Something went wrong" (see screenshots below).

## Steps to reproduce

You need to do this to reproduce this issue (naming and positions are from Samsung Galaxy S22):

1. Open app GitHub (tried: from Settings --> App --> Open, some Android launcher).
2. Tap the "Profile" button in the bottom menu (profile icon in the circle).
3. Tap settings icon in the top right corner (near share icon).
4. Scroll down the screen and tap on "Get Help".
5. Fill in the "Subject (required)" input field with "Test"
6. Fill in the "Body (required)" text area with "This is a test report"
7. Tap on the send icon in the top right corner.
8. Wait a few seconds since the error "Something went wrong" appears in the bottom of the screen.

## Expected behavior

Error report is sent successfully without any error message.

## Last known functional state

I don't remember, if I used this feature before. I think I haven't used it yet (at least on devices I currently have).

## Actions tried to resolve the issue
- Subject and message was minimized to "Test" and "This is a test report" respectively (without attached screenshot).
- App's cache was cleared (via Settings --> Apps --> GitHub --> Storage --> Clear cache).
- App's data was cleared (via Settings --> Apps --> GitHub --> Storage --> Clear data).
- Uninstall the app (via Settings --> Apps GitHub --> Uninstall) and installed again from Google Play store.
- Tested on completely different and independent (even different device type) device (same behavior) - see below

## SW/HW information

### First device (phone)

- **OS:** Android 13 (with OneUI 5.1), last update: 1st June 2023
- **App:** [GitHub](https://play.google.com/store/apps/details?id=com.github.android&hl=en&gl=US) (installed from Google Play) v1.126.0 (should be latest, Google Play don't offer newer version for me)
- **Device:** Samsung Galaxy S22 (SM-S901B/DS), bought in the Czech republic from Mironet
- **Kernel version:** 5.10.136-android12-9-26203352-abS901BXXS6CWGA (installed 25th July 2023 at 15.43.37)
- **Build number:** TP1A.220624.014.S901BXXS6CWGA
- **Knox version:** 3.9 (Knox ML 1.3, DualDAR 1.5.1, HDM 2.0)
- **Service provider software version:** SAOMC_SM-S901B_OXM_EUX_13_0164
- **Android security patch level:** 1st August 2023
- **Used IP addresses:** link-local IPv6, private IPv4 behind NAT, 2 public IPv6
- **Internet connection type:**
    - Wi-fi over dual 5 GHz and 2.4 GHz (beamed by [Mikrotik RBD52G-5HacD2HnD-TC](https://www.alza.cz/mikrotik-rbd52g-5hacd2hnd-tc-d5305635.htm))
    - 4G+ LTE (provided by O2 Czech Republic via primary SIM card)
- **Allow background usage** (in Settings --> Apps --> GitHub --> Mobile data): checked (`true`)
- **Allow data usage while Data saver is on** (in Settings --> Apps --> GitHub --> Mobile data): checked (`false`)
- **Battery optimization level** (in Settings --> Apps --> GitHub --> Battery): Optimized (default)
- **Data saver:** off

### Second device (tablet)

- **OS:** Android 13 (with OneUI 5.1), last update: 1st February 2023
- **App:** [GitHub](https://play.google.com/store/apps/details?id=com.github.android&hl=en&gl=US) (installed from Google Play) v1.126.0 (should be latest, Google Play don't offer newer version for me)
- **Device:** Samsung Galaxy Tab S8 (SM-X700), bought in the Czech republic from Samsung
- **Kernel version:** 5.10.136-android12-9-26317380-abX700XXU4BWD8 (installed 27th April 2023 at 08.51.30)
- **Build number:** TP1A.220624.014.X700XXU4BWD8
- **Knox version:** 3.9 (Knox ML 1.3, DualDAR 1.5.1, HDM 2.0)
- **Service provider software version:** SAOMC_SM-X700_OXM_EUX_13_0003
- **Android security patch level:** 1st May 2023
- **Used IP addresses:** link-local IPv6, private IPv4 behind NAT, 2 public IPv6
- **Internet connection type:**
    - Wi-fi over dual 5 GHz and 2.4 GHz (beamed by [Mikrotik RBD52G-5HacD2HnD-TC](https://www.alza.cz/mikrotik-rbd52g-5hacd2hnd-tc-d5305635.htm))
- **Allow background usage** (in Settings --> Apps --> GitHub --> Mobile data): checked (`true`)
- - **Allow data usage while Data saver is on** (in Settings --> Apps --> GitHub --> Mobile data): checked (`false`)
- **Battery optimization level** (in Settings --> Apps --> GitHub --> Battery): Optimized (default)
- **Data saver:** off

## Screenshots
I tried this pretty minimal report:
![Screenshot_20230822_195922_GitHub](https://github.com/community/community/assets/48548230/4939c160-ac31-48cb-87ff-f16ad9d119d1)

This is, how the error message looks like:
![Screenshot_20230822_202657_GitHub](https://github.com/community/community/assets/48548230/866506ee-3ae6-4ac3-9b3e-41d9aeb454b0)
![tempFileForShare_20230822-202749](https://github.com/community/community/assets/48548230/5d6177b8-5858-429b-8215-64a1e926c470)

## Warnings and cautions

I'm not a professional tester, so this is just a very amateur work of Master student at FIT BUT (in the 1st study year). There could be some important information missing, so you use this report at your own risk.
